bundle/language_en.properties

```diff
@@ -665,24 +665,56 @@ angal.hospital.visitduration.txt
+angal.inventory.chargetype.label                                                                       = Charge Type:
+angal.inventory.choosechargetypebeforevalidation.msg                                                   = Please choose the charge type and save the inventory before validation.
+angal.inventory.choosedestinationbeforevalidation.msg                                                  = Please choose the destination and save the inventory before validation.
+angal.inventory.choosedischargetypebeforevalidation.msg                                                = Please choose the discharge type and save the inventory before validation.
+angal.inventory.choosesupplierbeforevalidation.msg                                                     = Please choose the supplier and save the inventory before validation.
+angal.inventory.clean.btn                                                                              = Clean Table
+angal.inventory.clean.btn.key                                                                          = T
 angal.inventory.date.label                                                                             = Inventory's date
-angal.inventory.edit.title                                                                             = Edit an inventory
+angal.inventory.deletion.confirm.msg                                                                   = Do you really want to delete this inventory?
+angal.inventory.deletion.success.msg                                                                   = The inventory has been successfully marked as deleted.
+angal.inventory.deletion.error.msg                                                                     = Deletion is only allowed for inventories with 'draft' or 'validated' status.
+angal.inventory.destination.label                                                                      = Destination:
+angal.inventory.dischargetype.label                                                                    = Discharge Type:
+angal.inventory.done                                                                                   = Done
+angal.inventory.doyoureallywanttocleanthistable.msg                                                    = Do you really want to clean this table?
+angal.inventory.doyoureallywanttodeletethisinventoryrow.msg                                            = Do you really want to delete this inventory row?
+angal.inventory.doyouwanttoaddallnotyetlistedproducts.msg                                              = Do you want to add all not yet listed products?
+angal.inventory.doyouwanttocontinueediting.msg                                                         = Do you want to continue editing?
+angal.inventory.doyouwanttosavethechanges.msg                                                          = Do you want to save the changes?
+angal.inventory.draft                                                                                  = Draft
```

Comment:
Where is this used?

Reply:
It's used in the InventoryWardBrowser file on line 563